proto/minder/v1/minder.proto

```diff
@@ -1000,14 +1000,32 @@ message CheckHealthResponse {
 
 message GetAuthorizationURLRequest {
     bool cli = 3;
-    optional string owner = 5;
+    optional string owner = 5 [
+        (buf.validate.field).string = {
+            pattern: "[A-Za-z][-[:word:]]*"
+            max_len: 200,
+        },
+        (buf.validate.field).ignore = IGNORE_IF_DEFAULT_VALUE
+    ];
     Context context = 6;
-    optional string redirect_url = 7;
+    optional string redirect_url = 7 [
+        (buf.validate.field).string = {
+            pattern: "^(https?):\\/\\/[a-zA-Z0-9.-]+(\\.[a-zA-Z]{2,})?(:[0-9]{1,5})?(\\/[^\\s]*)?$",
```

Comment:
Why not use the `.string.uri` validation?

Reply:
Fixed it 👍